srcpkgs/xca/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'xca'
 pkgname=xca
-version=1.3.2
-revision=4
+version=2.0.1
+revision=1
 build_style=gnu-configure
-make_build_target="all"
-hostmakedepends="pkg-config qt5-tools qt5-host-tools"
+hostmakedepends="pkg-config qt5-tools qt5-host-tools perl"
 makedepends="qt5-devel libressl-devel libltdl-devel"
+depends="qt5-plugin-sqlite"
 short_desc="X Certificate and Key Management"
 maintainer="Denis Revin <denis.revin@gmail.com>"
 license="BSD"
```

Comment:
please use SPDX identifier

Reply:
as you wish, but Manual.md says: 
```license: A string matching any license file available in /usr/share/licenses. Multiple licenses should be separated by commas, i.e GPL-3, LGPL-2.1.```
maybe it must be changed if you want SPDX there?

Reply:
working on it.  https://github.com/voidlinux/void-packages/issues/12219